src/components/Header/Header.css

```diff
@@ -1,22 +1,30 @@
 .header {
-  display: flex;
-  justify-content: space-between;
+  display: grid;
+
   align-items: center;
+  justify-items: center;
+  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
restore previous changes and we will add a new icon to the header and the effect will be the same